config/shared.js

```diff
@@ -133,6 +133,18 @@ StyleDictionary.registerTransform({
   transformer: StyleDictionary.transform['color/css'].transformer,
 });
 
+StyleDictionary.registerTransform({
```

Comment:
This register call could live in `compose.config.js` since it's actually not shared with other configs.